electron/src/db/index.ts

```diff
@@ -14,31 +14,38 @@ export function closeDB(db: sqlite3.Database) {
   db.close();
 }
 
-export async function checkIfRecordsExist(
+export async function getRecordCounts(
   db: sqlite3.Database,
   checkQuery: string
-): Promise<boolean> {
+): Promise<number> {
   const checkResult = await sqlite3Wrapper.allP(db, checkQuery);
   if (checkResult && Number(checkResult[0].count) > 0) {
     log.info(`${checkResult[0].count} records found`);
-    return true;
+    return checkResult[0].count;
   }
   log.info(`${db}: No records found`);
-  return false;
+  return 0;
 }
 
-export async function returnDBIfPopulated(
+export interface DBRecordCount {
```

Comment:
DBWithRecordCount

Reply:
also why do we want record count

Reply:
we want record count to know the largest